src/Sylius/Bundle/CoreBundle/Form/Type/Grid/Filter/EntitiesFilterType.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Form\Type\Grid\Filter;
+
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @experimental
+ */
+final class EntitiesFilterType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
```

Comment:
Redundant docblock

Reply:
Kinda, but I guess we should remove them once and for all soon.